packages/account/src/Components/api-token/api-token-table-row-token-cell.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Clipboard, Text, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const ApiTokenTableRowTokenCell = ({ token }) => {
+    const [should_show_token, setShouldShowToken] = React.useState(false);
+
+    const toggleTokenVisibility = () => {
+        setShouldShowToken(!should_show_token);
```

Comment:
Since u r togging the value of the current state and setting it as the new state, may be it is better to use call back function and toggle the value of the previous state. Like `setShouldShowToken((prev_state)=>!prev_state);` This will ensure that u will be working with the latest value of the state

Reply:
Thanks for this remark. :+1: